doc/rados/configuration/mclock-config-ref.rst

```diff
@@ -4,19 +4,20 @@
 
 .. index:: mclock; configuration
 
-This dmclock related configuration parameters can be applied using mclock
-profiles. The purpose of the mclock profiles is to mask the low level details
-from a user trying to configure mclock for a specific purpose. At a high level,
-for each OSD in the enviroment the steps involve,
+Dmclock-related configuration parameters can be applied by using mclock
+profiles. Mclock profiles mask the low level details from users, making it
+easier for them to configure mclock. These dmclock-related configuration
+parameters do the following: 
 
-- providing an input related to the total capacity of OSD(s) and
-- choosing the desired mclock profile to enable
+- provide an input related to the total capacity of OSD(s) and
+- choose an mclock profile to enable
 
-Based on the profile specified, the OSD determines and applies the lower level
-mclock and Ceph parameters. The parameters applied by the mclock profile
-enables tuning of the QoS between client, background recovery/backfill ops and
-other internal entities (aka best effort clients) within Ceph that generate IOPS
-(for e.g. scrubs, snap trim, PG deletions etc.).
+Using the settings in the specified profile, the OSD determines and applies the
+lower-level mclock and Ceph parameters. The parameters applied by the mclock
+profile make it possible to tune the QoS between client, background
+recovery/backfill ops, and other internal entities (for example, best effort
```

Comment:
here "internal entities" implies scrubs, snap trim, and PG deletions.

how about something like "QoS between client I/O, recovery/backfill operations, and other background operations (for example scrub, snap trim, and PG deletion). These background activities are considered best effort internal clients of Ceph."?

Reply:
This has been added.